core/controllers/admin_test.py

```diff
@@ -690,7 +688,7 @@ def test_cannot_view_role_with_invalid_view_method(self):
 
         self.assertEqual(response['error'], 'Invalid method to view roles.')
 
-    def test_deassign_user_from_all_topics(self):
+    def test_assign_moderator(self):
```

Comment:
Try to be more descriptive in test names please. E.g.

test_changing_user_role_from_topic_manager_to_moderator

Reply:
done